# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1316405326 25200
#      Sun Sep 18 21:08:46 2011 -0700
# Node ID a8184fb6b0c7bd773e58fe3d61e31fdfd77b0426
# Parent  a1049e4480f8fa3fd597ff4375314d351a30f1e3
strtrim.m: Rename function inside .m file to match filename.

* strtrim.m: Rename function inside .m file to match filename.

diff --git a/scripts/strings/strtrim.m b/scripts/strings/strtrim.m
--- a/scripts/strings/strtrim.m
+++ b/scripts/strings/strtrim.m
@@ -34,17 +34,17 @@
 ## @end example
 ## @seealso{deblank}
 ## @end deftypefn
 
 ## Author: John Swensen <jpswensen@jhu.edu>
 
 ## This function was derived from deblank.
 
-function s = strtrim2 (s)
+function s = strtrim (s)
 
   if (nargin != 1)
     print_usage ();
   endif
 
   if (ischar (s))
 
     k = find (! isspace (s));
